client/payment-details/summary/style.scss

```
@@ -19,6 +19,22 @@
 		flex-grow: 1;
 	}
 
+	.payment-details-summary__amount_wrapper {
+		display: flex;
+		align-items: center;
+	}
+
+	@media screen and ( max-width: $break-small ) {
+		.payment-details-summary__amount_wrapper {
+			flex-direction: column;
+			align-items: flex-start;
+
+			.payment-details-summary__status {
+				order: -1;
+			}
+		}
+	}
```

⛏️ Similarly, for consistency with BEM, I suggest we use `-`  in `amount-wrapper` rather than `_`.

```suggestion
	.payment-details-summary__amount-wrapper {
		display: flex;
		align-items: center;
	}

	@media screen and ( max-width: $break-small ) {
		.payment-details-summary__amount-wrapper {
			flex-direction: column;
			align-items: flex-start;

			.payment-details-summary__status {
				order: -1;
			}
		}
	}
```

---

Address in [13db0c1](https://github.com/Automattic/woocommerce-payments/pull/9790/commits/13db0c1a5a9f0c2336a9da3a796ed31e541c86d7).